modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides the support for legacy BIOS backward compatibility with UEFI systems.
```

Comment:
You could include a mention to Fujitsu iRMC in this introductory paragraph, to give the reader context about how this section is explicit to iRMC use cases. I can determine from the heading that the section relates to  Fujitsu iRMC, but further context would be helpful in the introduction.

Reply:
How about moving the NOTE to this paragraph and delete the NOTE?  For example:
> The Compatibility Support Module (CSM) configuration provides support for legacy BIOS backward compatibility with UEFI systems. You must configure the CSM when you deploy a cluster with Fujitsu iRMC, otherwise the installation might fail.

Reply:
That looks good and provides the necessary context in the paragraph.